src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvOrganizationEntityUpdater.java

```diff
@@ -136,20 +127,27 @@ public void onOrganizationIdRemoveRequest(@Observes (during = TransactionPhase.B
   @Timeout
   public void timeout(Timer timer) {
     if (!stopped) {
-      if (!queue.isEmpty()) {
-        updateOrganization(queue.remove(0));
+      OrganizationIdUpdateRequest updateRequest = queue.next();
+      if (updateRequest != null) {
+        updateOrganization(updateRequest);
       }
 
       startTimer(SystemUtils.inTestMode() ? 1000 : TIMER_INTERVAL);
     }
   }
 
-  private void updateOrganization(OrganizationId organizationId) {
+  private void updateOrganization(OrganizationIdUpdateRequest updateRequest) {
+    updateOrganization(updateRequest.getId(), updateRequest.getOrderIndex());
+  }
+
+  private void updateOrganization(OrganizationId organizationId, Long orderIndex) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "updateOrganization" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)